client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/index.tsx

```diff
@@ -63,26 +64,38 @@ const LinkInBioSetup: Step = function LinkInBioSetup( { navigation } ) {
 
 	const handleSubmit = async ( event: FormEvent ) => {
 		event.preventDefault();
-		if ( site ) {
-			await saveSiteSettings( site.ID, {
-				blogname: siteTitle,
-				blogdescription: tagline,
-			} );
-			recordTracksEvent( 'calypso_signup_site_options_submit', {
-				has_site_title: !! siteTitle,
-				has_tagline: !! tagline,
-			} );
-
-			if ( selectedFile ) {
-				try {
-					await setSiteLogo( selectedFile );
-				} catch ( _error ) {
-					// communicate the error to the user
+
+		setPendingAction( async () => {
+			if ( site ) {
+				setProgress( 0 );
+				setProgressTitle( __( 'Turning on the lights' ) );
```

Comment:
I think this is supposed to cycle through a bunch of sayings like in this hook ⇢ https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts

We could move that hook to the main /hook folder and make it functional for both steps and potentially more in the future?